dev-client/src/translations/en.json

```diff
@@ -9,8 +9,13 @@
     "members": "{{members}} Members",
     "show_on_map": "SHOW ON MAP",
     "create": "CREATE SITE",
+    "more_info": "MORE INFO",
     "none_existing_p1": "You don’t currently have any saved sites. Create a site by using the button above or by finding the site location on the map.",
-    "none_existing_p2": "A site allows you to collect data, learn more about your land, and share with others. Learn more about using LandPKS..."
+    "none_existing_p2": "A site allows you to collect data, learn more about your land, and share with others. Learn more about using LandPKS...",
```

Comment:
```suggestion
    "none_existing_p2": "A site allows you to collect data, learn more about your land, and share with others. Learn more about using LandPKS.",
```

Reply:
i am following the design for this wording, so i think feedback makes more sense to direct there